### ISSUE TITLE
Development of interactive games

### ISSUE THREAD

**Issue by dumblob** [Wicklets/wick-editor#293]
**Is your feature request related to a problem? Please describe.**
I'm curious whether Wick can be used for development of small and medium interactive games.

What is missing in Wick to implement the 25 games from https://beebom.com/browser-games/ ?

Would it be wise to implement these games in Wick?

Which of these games would be less difficult to implement than others and why?

Does development of such games match with Wick's goals and audience?

How hard would it be to implement missing functionality in Wick to support development of such games?

**Describe the solution you'd like**
A non-programmer would prepare 70% of the content (by clicking, drawing, animating, layouting, etc.), then give it over to a programmer who would add some "business logic" (20%) and they both would meet to do the rest 10% together.

Basically an *exploratory* approach instead of creating an up-front specification first and then developing according to such document.

**Describe alternatives you've considered**
Using a programming game engine with the traditional approach (i.e. concept art, then story making, then development with integration in bi-weekly iterations, then beta release, then first release).

**Additional context**
The goal is to create small games with much less effort and with low technical knowledge. Imagine kids or creative/artistic woman.

**Comment by sporklpony** [comment]
Some of those games are 3d which Wick isn't really designed to do, and multiplayer can be difficult to implement (but my understanding is that it should be possible, but one would need to do a lot of JS plumbing and would need to write server code separately), but my understanding is that the goal of Wick is to make game development fast and easy, and not require a ton of technical knowledge, as you describe. Many of the inspirations for Wick are famous for not requiring technical knowledge. You've probably already seen it, but if you haven't, do check out the Community page on the Wick website (https://www.wickeditor.com/#/community/) for some examples of projects made with Wick.
